components/Team/TeamWorkList.tsx

```diff
@@ -1,42 +1,123 @@
-        <Accordion.Item eventKey={id!} key={id}>
-          <Accordion.Header>
-            {title} - {updatedAt ? updatedAt.slice(0, 10) + ' 更新' : ''}
-          </Accordion.Header>
-          <Accordion.Body>
-            <p>{description}</p>
-            {type === TeamWorkType.IMAGE ? (
-              <Image src={url} className="mw-100" alt={title} />
-            ) : type === TeamWorkType.VIDEO ? (
-              <Ratio aspectRatio="16x9">
-                <video controls width="250" src={url} />
-              </Ratio>
-            ) : (
-              <a href={url} title={title}>
-                {title}
-              </a>
-            )}
-          </Accordion.Body>
-        </Accordion.Item>
-      ))}
+      <Link href={`/activity/${activity}/team/${team}/work/create`}>
+        <Button variant="success" className="me-3 mb-2">
+          提交作品
+        </Button>
+      </Link>
+      <Row
+        className="g-4"
+        xs={1}
+        sm={2}
+        {...(size === 'sm'
+          ? {}
+          : !size
+          ? { lg: 3, xxl: 4 }
+          : { lg: 4, xxl: 6 })}
+      >
+        {value.map(({ updatedAt, id, title, description, type, url }) => (
+          <Col key={id}>
+            <Card className="border-success">
+              <Card.Body>
+                <Card.Title
+                  className="text-primary text-truncate"
+                  title={title}
+                >
+                  <Link
+                    href={`/activity/${activity}/team/${team}/work/${id}/edit`}
+                  >
+                    <a>{title}</a>
+                  </Link>
+                </Card.Title>
+                <Row className="border-bottom py-2 g-4">
+                  <span className="text-muted text-truncate">
+                    {description}
+                  </span>
+                </Row>
+                <Row className="border-bottom py-2 g-4">
+                  {type === TeamWorkType.IMAGE ? (
+                    <Image src={url} className="mw-100" alt={title} />
+                  ) : type === TeamWorkType.VIDEO ? (
+                    <Ratio aspectRatio="16x9">
+                      <video controls width="250" src={url} />
+                    </Ratio>
+                  ) : (
+                    <a
+                      className="text-primary"
+                      target="_blank"
+                      href={url}
+                      title="查看作品"
+                      rel="noreferrer"
+                    >
+                      查看作品
+                    </a>
+                  )}
+                </Row>
+                <Row as="small" className="border-bottom py-2 g-4">
+                  <Col className="text-truncate" title="更新时间">
+                    <FontAwesomeIcon
+                      className="text-success me-2"
+                      icon={faCalendarDay}
+                    />
+                    {updatedAt}
+                  </Col>
+                </Row>
```

Comment:
```suggestion
                <time className="border-bottom p-2 text-truncate" title="更新时间" dateTime={updatedAt}>
                    <FontAwesomeIcon
                      className="text-success me-2"
                      icon={faCalendarDay}
                    />
                    {new Date(updatedAt).toLocaleString()}
                </time>
```
1. 同上
2. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time
3. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString